src/@types/expenseListsResponse

```diff
@@ -0,0 +1,12 @@
+import { ExpenseLIstType } from "./expense-list-prop";
+import { SortOrder } from "./sortOrderTypes";
+
+export interface ExpenseListsResponse {
+  data: ExpenseLIstType[];
+  isLoading: boolean;
+  error: Error | null;
```

Comment:
use error?: Error instead of Error | Null

Reply:
fixed